src/app/interfaces/bind-ip.interface.ts

```diff
@@ -1,3 +1,3 @@
 export interface BindIp {
```

Comment:
`BindIp` is only used in `service-smb.component.ts` - we may remove it from global interfaces.